sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs

```diff
@@ -25,6 +25,7 @@ public void ApplyToTest(Test test)
             if (test.RunState != RunState.NotRunnable)
             {
                 RecordedTestMode mode = RecordedTestUtilities.GetModeFromEnvironment();
+                mode = RecordedTestMode.Live;
```

Comment:
??

Reply:
Thank you!!! left after some testing. Reverting ....